packages/app-builder-lib/src/electron/ElectronFramework.ts

```diff
@@ -97,27 +97,29 @@ async function removeUnusedLanguagesIfNeeded(options: BeforeCopyExtraFilesOption
     return
   }
 
-  const { dir, langFileExt } = getLocalesConfig(options)
+  const { dirs, langFileExt } = getLocalesConfig(options)
   // noinspection SpellCheckingInspection
-  await asyncPool(MAX_FILE_REQUESTS, await readdir(dir), async file => {
-    if (!file.endsWith(langFileExt)) {
-      return
-    }
+  const deletedFiles = async (dir: string) => {
+    await asyncPool(MAX_FILE_REQUESTS, await readdir(dir), async file => {
+      if (!file.endsWith(langFileExt)) {
+        return
+      }
 
-    const language = file.substring(0, file.length - langFileExt.length)
-    if (!wantedLanguages.includes(language)) {
-      return fs.rm(path.join(dir, file), { recursive: true, force: true })
-    }
-    return
-  })
+      const language = file.substring(0, file.length - langFileExt.length)
```

Comment:
I realize this isn't new code, but I think we could do here:
```
path.extname(file) !== langFileExt // if we remove the `.` from the extension filter
```
AND
```
const language = path.basename(file, langFileExt)
```

Reply:
Fixed.